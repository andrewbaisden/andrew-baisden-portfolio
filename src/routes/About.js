import React from 'react';
import { Helmet } from 'react-helmet';
import Header from '../components/Header';
import Footer from '../components/Footer';
import ProfileImage from '../components/ProfileImage';
import AboutPod from '../components/AboutPod';
import CheckAboutPage from '../components/CheckAboutPage';
import ServicesBreadcrumb from '../components/ServicesBreadcrumb';

const Home = () => (
  <div>
    <div>
      <Helmet>
        <meta
          name="description"
          itemProp="description"
          content="I’m Andrew thanks for checking out my website! My career choice was to be a Developer. I have always had a passion for programming. For those of you who are not technical on this subject, it basically means that I’m a wizard! Well, maybe not a real wizard but the things I can do, really can be perceived as magic!"
        />
        <title>About - Full Stack Developer | Andrew Baisden | London</title>
        <link rel="canonical" href="https://andrewbaisden.com/about" />
      </Helmet>
    </div>
    <CheckAboutPage />
    <Header />
    <ProfileImage />
    <ServicesBreadcrumb />
    <AboutPod />
    <Footer />
  </div>
);

export default Home;
