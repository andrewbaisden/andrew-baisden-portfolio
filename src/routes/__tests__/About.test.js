import React from 'react';
import { shallow } from 'enzyme';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import About from '../About';

Enzyme.configure({ adapter: new Adapter() });

describe('<About />', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<About />);
    expect(wrapper).toMatchSnapshot();
  });
});
