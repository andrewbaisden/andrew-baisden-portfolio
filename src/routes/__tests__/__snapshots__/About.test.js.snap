// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> renders and matches the snapshot 1`] = `ShallowWrapper {}`;
